### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by duncandewhurst**
For background, see https://github.com/open-contracting/ocds-extensions/issues/201 and [Slack discussion](https://opencontractingint.slack.com/archives/C03S8JL100P/p1681773662849569?thread_ts=1661872262.757289&cid=C03S8JL100P).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,7 @@ In the European Union, this extension's fields correspond to the following [eFor
 
 * BT-660 (Framework Re-estimated Value)
 * BT-709 (Framework Maximum Value)
-* BT-150 (Contract identifier)
 * BT-708 (Documents Official Language)
-* BT-67 (Exclusion Grounds)
-* BT-22 (Internal Identifier)
 
 For correspondences to eForms fields, see OCDS for eForms.
```
Comment (odscjen) [inline README.md]:
```suggestion
For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/).
```
I know the link is already in the first section but for completeness it feels like it should be repeated here

**Review by odscjen — CHANGES_REQUESTED** [review]
There's no change log at the bottom of the README.md, should there be to reference the removal of `identifiers` and `exclusionGrounds`?

**Comment by duncandewhurst** [comment]
> There's no change log at the bottom of the README.md, should there be to reference the removal of `identifiers` and `exclusionGrounds`?

I don't think so as we're still authoring the initial version of the extension.

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
Do you remember why we split out simple identifier into its own extension? I can't remember now.

I think maybe we wanted to reuse the SimpleIdentifier definition in other extensions, but I can't remember which.

**Comment by duncandewhurst** [comment]
It was so that the lots extension could depend on it without needing to depend on the whole of the 1.2 draft extension: https://github.com/open-contracting-extensions/ocds_lots_extension/pull/41#discussion_r1160273306

**Comment by duncandewhurst** [comment]
Happy for this to be merged?

**Comment by jpmckinney** [comment]
> It was so that the lots extension could depend on it without needing to depend on the whole of the 1.2 draft extension: https://github.com/open-contracting-extensions/ocds_lots_extension/pull/41#discussion_r1160273306

Oh, right – but later, we learned that all dependencies need to be explicitly declared in `extensions` arrays, because the Data Review Tool and other tools don't apply dependencies on their own.

So, in the end, we added SimpleIdentifier directly to the lots extension, since the changes to lots was breaking existing publications.

As such, we maybe don't want to remove these from this extension, and we can delete the identifiers extension (and merge back its updates into this extension).

It's a bit of a run-around, but at least we learned/reminded ourselves of how dependencies work.

**Comment by duncandewhurst** [comment]
All good :-) I'll tackle those changes next week.

**Comment by duncandewhurst** [comment]
@jpmckinney I've archived https://github.com/open-contracting-extensions/ocds_tender_contract_identifiers_extension. I don't have permission to delete it.

**Comment by jpmckinney** [comment]
Thanks - I've removed it from the extension registry as well.

**Pull request closed without merging**